cclib/parser/orcaparser.py

```diff
@@ -796,9 +795,13 @@ def splitter(line):
                         contrib = numpy.nan
                     sec.append([start, end, contrib])
                     line = next(inputfile)
-                self.etsecs.append(sec)
+                etsecs.append(sec)
                 line = next(inputfile)
 
+            self.etenergies = etenergies
+            self.etsecs = etsecs
+            self.etsyms = etsyms
```

Comment:
Maybe use `set_attribute` for these three?

Reply:
Done, and in several other places.